src/gateway_config_worker.erl

```diff
@@ -218,6 +218,7 @@ handle_info({packet, Packet}, State=#state{}) ->
 handle_info({button_clicked, _, 1}, State=#state{}) ->
     lager:info("Button clicked"),
     %% Start a scan for visible wifi services
+    connman:enable(wifi, true),
```

Comment:
So if the enable takes a bit or fails, the scan request on the next line won't start at all right?

Reply:
That's right. This is a Hail Mary pass that seemed to help before I added the retries. Enabling Wi-Fi when it's disabled does a scan as well.  Enabling Wi-Fi when it's already enabled appears to do nothing.

```
connmanctl> enable wifi
Error wifi: Already enabled
connmanctl>
```